### DISCUSSION TITLE
`extend_to_end: false`  not supported ?

### DISCUSSION THREAD

**Discussion by StorageGuru** [RomRider/apexcharts-card#563] [Q&A]
I want to show a graph for a 24h period, but values after the actual time should not be shown in the graph.
I found some entries here pointing to 

`extend_to_end: false` 

but that does not seem to work. Any ideas how I can stop the card showing non-sense values after "now" ?

here is my yaml code for the card:

`type: custom:apexcharts-card
experimental:
  color_threshold: true
hours_12: false
span:
  start: day
header:
  show: true
  title: Strom Bezug/Erzeugung heute
  show_states: true
  colorize_states: true
now:
 show: true
 color: red
 label: Now
series:
  - entity: sensor.strom_aktuell
    type: line
    **extend_to_end: false** 
    group_by:
      func: last
      duration: 15m
    stroke_width: 2
    show:
      header_color_threshold: true
    color_threshold:
      - value: 0
        color: green
      - value: 0
        color: red`

**Comment by hofstee** [comment]
I'm fairly new to this card, but have you tried `extend_to: false` instead? That seems to be what the docs suggest: https://github.com/RomRider/apexcharts-card#series-options

**Reply by StorageGuru** [reply]
Hi hofstee, thank you. Your hint led me to the solution.  

Seems the doc has been updated:

`extend_to_end: false` is deprecated but I can now use:

`extend_to: now` 

This only extends the graph to the current time.
